pkg/kapp/cmd/app/delete.go

```diff
@@ -182,6 +182,14 @@ func (o *DeleteOptions) calculateAndPresentChanges(existingResources []ctlres.Re
 			return ctlcap.ClusterChangeSet{}, nil, false, err
 		}
 
+		// Apply diff-filter on the changes
```

Comment:
nit: comment does not say anything that the code is not doing.

Reply:
Yeah removed.